cfg/cis-1.9/master.yaml

```diff
@@ -189,7 +189,7 @@ groups:
       - id: 1.1.13
         text: "Ensure that the default administrative credential file permissions are set to 600 (Automated)"
         audit: |
-         for adminconf in /etc/kubernetes/{admin.conf,super-admin.conf}; do if test -e $adminconf; then stat -c \"permissions=%a %n\" $adminconf; fi; done
+         for adminconf in /etc/kubernetes/{admin.conf,super-admin.conf}; do if test -e $adminconf; then stat -c "permissions=%a %n" $adminconf; fi; done
```

Comment:
i do not think this will work because curly brace expansion is a feature of bash and this is run with sh

https://github.com/aquasecurity/kube-bench/blob/f8b6f2fc19565e1c9b33f2548e9e7a75dc195488/check/check.go#L299

for example

```
ethan@ethanm-kurl-1:~$ echo 'for adminconf in /etc/kubernetes/{admin.conf,super-admin.conf}; do if test -e $adminconf; then stat -c "ownership=%U:%G %n" $adminconf; fi; done' | sh
ethan@ethanm-kurl-1:~$ echo 'for adminconf in /etc/kubernetes/{admin.conf,super-admin.conf}; do if test -e $adminconf; then stat -c "ownership=%U:%G %n" $adminconf; fi; done' | bash
ownership=root:root /etc/kubernetes/admin.conf
ownership=root:root /etc/kubernetes/super-admin.conf
```